lib/auth/accountrecovery_test.go

```diff
@@ -1334,3 +1378,25 @@ func getMockedU2FAndRegisterRes(authSrv *Server, tokenID string) (*proto.U2FRegi
 		ClientData:       u2fRegResp.ClientData,
 	}, u2fKey, nil
 }
+
+func getMockedU2FAndRegisterRes(authSrv *Server, tokenID string) (*wantypes.CredentialCreationResponse, *mocku2f.Key, error) {
+	res, err := authSrv.CreateRegisterChallenge(context.TODO(), &proto.CreateRegisterChallengeRequest{
+		TokenID:    tokenID,
+		DeviceType: proto.DeviceType_DEVICE_TYPE_WEBAUTHN,
+	})
+	if err != nil {
+		return nil, nil, trace.Wrap(err)
+	}
+
+	u2fKey, err := mocku2f.Create()
+	if err != nil {
+		return nil, nil, trace.Wrap(err)
+	}
```

Comment:
Suggestion: just call it key, so we don't overload readers with the U2F/Webauthn nomenclature?

Also, you should set PreferRPID = true here, you may have problems otherwise. (Or, better yet, use the TestDevice API.)

```suggestion
	key, err := mocku2f.Create()
	if err != nil {
		return nil, nil, trace.Wrap(err)
	}
	key.PreferRPID = true
```